db/migrate/20160708204023_remove_last_replication_colulmns_from_miq_database.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveLastReplicationColulmnsFromMiqDatabase < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
I believe you can simplify this with `def change` and specify the column type on the remove_column